templates/avatars.php

```diff
@@ -0,0 +1 @@
+not yet implemented. planned: a grid of all ContactDetailAvatars of contacts in that addressbook which have a social profile and can be updated. and an update-all-button, of course!
```

Comment:
So, I don't want to go this far.
I like the idea of an underlying sync system for avatars, but I really don't want to turn the app into a sync manager :)

Reply:
OK - let's do it as suggested below

Reply:
I am ready to tackle background jobs... should I wait for #1594 or is there anything I can already prepare?

Reply:
Let's do background job in a different pr :stuck_out_tongue:

Reply:
alright. So what needs to be done to integrate this pull request? Besides the sync icon, of course. :laughing: